doc-coverage/raco.rkt

```diff
@@ -0,0 +1,101 @@
+#lang racket
+
+(require raco/command-name
+         racket/cmdline
+         rackunit/docs-complete
+         "main.rkt")
+
+; do-ignore! : (U string symbol) regex -> boolean
+(define (do-ignore! mod ignore)
+  (error "Ignore not implemented yet")
+  #f
+  #;(define missing
+    (with-output-to-string
+      (lambda ()
+        (parameterize ([current-error-port (current-output-port)])
+          (check-docs mod #:skip ignore)))))
+  #;(match missing
+    ["" (printf "Module ~a is documented~n" a) #t]
+    [else (printf "Module ~a is missing documentation for ~a~n" a missing) #f]))
+
+; do-binding! : (U string symbol) symbol -> boolean
+(define (do-binding! mod binding)
+  (cond [(set-member? (module->all-exported-names mod) binding)
+         (define b* (has-docs? mod binding))
+         (cond [b* (printf "Module ~a has documentation for ~a~n" mod binding)
+                   #t]
+               [else (printf "Module ~a is missing documentation for ~a~n" mod binding)
+                     #f])]
+        [else
+         (fprintf (current-error-port) "Module ~a does not export ~a~n" mod binding)
+         #f]))
+
+; try-namespace-require : string -> (U 'string 'symbol #f)
+(define (try-namespace-require mod)
```

Comment:
If `try-namespace-require` returned the module name it ended up using or `#f`, the main loop body could be much better organized.